cwms-data-api/src/test/resources/cwms/cda/data/dto/location/kind/base_outlet.json

```diff
@@ -1,19 +1,26 @@
 {
   "project-id": {
     "office-id": "SPK",
-    "name": "location"
+    "name": "BIGH"
   },
   "location": {
     "office-id": "SPK",
-    "name": "outlet",
+    "name": "TG2",
     "latitude": 0.0,
     "longitude": 0.0,
     "active": true,
-    "public-name": "outlet",
+    "public-name": "TG2",
     "timezone-name": "UTC",
     "location-kind": "Outlet",
     "horizontal-datum": "NAD84",
     "vertical-datum": "NAVD88"
   },
-  "rating-group-id": "Rating-location-outlet"
+  "rating-category-id": {
+    "office-id": "SPK",
+    "name": "Rating"
+  },
+  "rating-group-id": {
```

Comment:
Include the rating specification id here as either a rating group specific key value or as the loc group's shared_loc_alias_id key with the rating spec id as value.

Reply:
Zack added the rating spec id value here.
https://github.com/USACE/cwms-data-api/blob/develop/cwms-data-api/src/test/resources/cwms/cda/data/dto/location/kind/base_outlet.json#L26